client/state/comments/selectors/comments-fetching-status.js

```diff
@@ -10,7 +9,7 @@ export const commentsFetchingStatus = createSelector(
 	( state, siteId, postId, commentTotal = 0 ) => {
 		const fetchStatus =
 			state.comments.fetchStatus[ getStateKey( siteId, postId ) ] ?? fetchStatusInitialState;
-		const hasMoreComments = commentTotal > size( getPostCommentItems( state, siteId, postId ) );
+		const hasMoreComments = commentTotal > getPostCommentItems( state, siteId, postId ).length;
```

Comment:
We might need some optional chaining here:

```suggestion
		const hasMoreComments = commentTotal > getPostCommentItems( state, siteId, postId )?.length;
```

because that selector could potentially return `undefined`:

https://github.com/Automattic/wp-calypso/blob/a14b6727ffc1ed68b951c95f8a43ca6e1cb15ccc/client/state/comments/selectors/get-post-comment-items.js#L12

Furthermore, `> undefined` doesn't make much sense either:

```
> 1 < undefined // false
> 1 > undefined // false
```

So we might want to default to `0`:

```suggestion
		const hasMoreComments = commentTotal > ( getPostCommentItems( state, siteId, postId )?.length ?? 0 );
```